GiwazipClient/Views/PostingPhotoViewController.swift

```diff
@@ -24,77 +46,171 @@ class PostingPhotoViewController: BaseViewController {
+    private func setupPHPickerConfigure() {
+        pickerConfiguration.selection = .ordered
+        pickerConfiguration.selectionLimit = (6 - images.count)
+        pickerConfiguration.filter = .any(of: [.images, .not(.livePhotos)])
+    }
+
+    private func showPHPicker() {
+        let picker = PHPickerViewController(configuration: pickerConfiguration)
+        picker.delegate = self
+        present(picker, animated: true)
+    }
+    
+    private func didTapChangeAction() {
+        isChangedConfigure = false
+        pickerConfiguration.selectionLimit = 1
+        pickerConfiguration.selection = .default
+        showPHPicker()
+        
+        images.remove(at: selectedIndex)
+    }
+    
+    private func didTapDeleteAction() {
+        photoCollectionView.reloadData()
+        images.remove(at: selectedIndex)
+    }
 }
 
 extension PostingPhotoViewController: UICollectionViewDelegate, UICollectionViewDataSource, UICollectionViewDelegateFlowLayout {
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        // TODO: - 이미지 갯수
-        return 1
+        return images.count
     }
-    
+
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         let cell = collectionView.dequeueReusableCell(withReuseIdentifier: PostingPhotoCell.identifier, for: indexPath) as! PostingPhotoCell
-        
+
+        cell.plusIcon.isHidden = (indexPath.item == 0 && images[0] == emptyImage) ? false : true
+
+        if (images.count == 6) && (images[0] == emptyImage) {
+            images.remove(at: 0)
+            photoCollectionView.reloadData()
+        } else {
+            cell.postingImage.image = self.images[indexPath.item]
+
+            if images.count < 5 && !images.contains(emptyImage) {
+                images.insert(emptyImage, at: 0)
+                photoCollectionView.reloadData()
+            }
+        }
+
         return cell
     }
-    
+
     func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
-        
         let width = UIScreen.main.bounds.width - 32
         let height = width / 4 * 3
-        
+
         return CGSize(width: width, height: height)
     }
-    
+
+    func collectionView(_ collectionView: UICollectionView, didSelectItemAt indexPath: IndexPath) {
+        self.selectedIndex = indexPath.item
+
+        setupPHPickerConfigure()
+
+        if (selectedIndex == 0) && (images[0] == emptyImage) {
+            isChangedConfigure = true
+            showPHPicker()
+        } else {
+            makeActionSheet(
+                firstContext: "사진 변경하기",
+                secondContext: "사진 삭제하기",
+                didTapFirst: { change in
+                    self.didTapChangeAction()
+                },
+                didTapSecond: { delete in
+                    self.didTapDeleteAction()
```

Comment:
p5;
self.didTapDeleteAction() 이후에 isChangedConfigure = true 를 해주면 좋지 않을까? 라는 생각이 듭니다. 
```suggestion
                    self.didTapDeleteAction()
                    isChangedConfigure = true
```

Reply:
> p5; self.didTapDeleteAction() 이후에 isChangedConfigure = true 를 해주면 좋지 않을까? 라는 생각이 듭니다.

그렇게 하게 되면, ActionSheet를 띄움과 동시에 isChangedConfigure가 True로 바뀌어, 이미지가 반영되는 위치에서 문제가 생겨 기존 코드로 유지하였습니다.

Reply:
> > p5; self.didTapDeleteAction() 이후에 isChangedConfigure = true 를 해주면 좋지 않을까? 라는 생각이 듭니다.
> 
> 그렇게 하게 되면, ActionSheet를 띄움과 동시에 isChangedConfigure가 True로 바뀌고, 이미지가 반영되는 위치에서 문제가 생겨 기존 코드로 유지하였습니다.

으악! 이건 제가 잘못생각한 것 같네요!

++ 여담이지만 제가 확인해본 결과, { delete in ~} 에서 뒤쪽에 있는 구문들은, 액션시트가 올라오고 해당 delete를 클릭하면 실행됩니다!!